### PR TITLE
Clarify onbuild image description

### PR DESCRIPTION
**Pull request by naamunds**
cc: @MichaelSimons @dleeapho

### CODE REVIEW

File: 1.0.0/README.md [hunk 1]
```diff
@@ -49,11 +49,7 @@ applications. Use this image for your development process (developing, building
 
 ### `microsoft/dotnet:<version>-onbuild`
 
-The most straightforward way to use this image is to use a .NET container as both the build and runtime environment. In your `Dockerfile`, writing something along the lines of the following will compile and run your project:
-
-```dockerfile
-FROM microsoft/dotnet:onbuild
-```
+The most straightforward way to use this image is to use a Docker container as both the build and runtime environment for your application. A `Dockerfile` in the same directory as your project files which uses this image as its parent will compile and run your project.
```
Comment (MichaelSimons) [inline 1.0.0/README.md]: What do you think about calling out more that the user's Dockerfile only needs the FROM? 

Creating a simple `Dockerfile` with the following content in the same directory as your project files will compile and run your project
```
FROM microsoft/dotnet:onbuild
```

Comment (naamunds) [inline 1.0.0/README.md]: Good idea. I've made that change.

**Comment by MichaelSimons** [comment]
:shipit:

**Comment by dleeapho** [comment]
LGTM

**Pull request merged**